src/common/SearchInput.js

```diff
@@ -44,7 +44,7 @@ export default function SearchInput(props: Props): Node {
         selectTextOnFocus
         underlineColorAndroid="transparent"
         autoCapitalize="none"
-        placeholder="Search"
+        placeholder="Search topics"
```

Comment:
This changes the placeholder text everywhere `SearchInput` is used, not just in the place we want to update. 😉 The issue #5038 just says "the search bar for topics in the stream topic view".

Reply:
Got it!